ambry-messageformat/src/main/java/com.github.ambry.messageformat/MessageFormatRecord.java

```diff
@@ -56,6 +57,8 @@
 
   static short headerVersionToUse = Message_Header_Version_V2;
 
+  private static final short Delete_Version_V1 = 1;
```

Comment:
Do you think it is easier to reason about things if we start with `Delete_Version_V3` (since `Delete_Version_V1` and `Delete_Version_V2` were used in the past as standalone record versions)? Basically, any mention about `Delete_Version_V1` in any discussion would be unambiguous that way.

Otherwise, how about calling this `Delete_Subrecord_Version_V1`?

Reply:
I'll take the second suggestion